ganga/GangaCore/Lib/BackendScriptTemplate.py.template

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import shutil
+import os
+import time
+import popen2
+import glob
+import mimetypes
+import sys
+import tarfile
+import subprocess
+import os,os.path,shutil,tempfile
+
+if len(sys.argv)>1 and sys.argv[1] == 'subprocess':
+    os.setsid()
+
+###INLINEMODULES###
+###INLINEHOSTNAMEFUNCTION###
+
+input_sandbox = ###INPUT_SANDBOX###
+sharedoutputpath = ###SHAREDOUTPUTPATH###
+outputpatterns = ###OUTPUTPATTERNS###
+execmd = ###APPSCRIPTPATH###
+environment = ###ENVIRONMENT###
+
+# jobid is a string
+jobid = ###JOBID###
+Backend = ###BACKEND###
+
+###PREEXECUTE###
+
+def flush_file(f):
+    f.flush()
+    os.fsync(f.fileno()) #this forces a global flush (cache synchronization on AFS)
+
+def open_file(fname):
+    try:
+        filehandle=open(fname,'w')
+    except IOError as x:
+        print('ERROR: not able to write a status file: ', fname)
+        print('ERROR: ',x)
+        raise
+    return filehandle
+
+def failurereport(filehandle, message):
+    errfile = open('stderr', 'w' )
+    errfile.close()
+    print('EXITCODE: %d' % -9999, file=filehandle)
+    print('FAILED: %s'%time.strftime('%a %b %d %H:%M:%S %Y'), file=filehandle)
+    print('PROBLEM STARTING THE APPLICATION SCRIPT: \'%s\' \'%s\''%(execmd,str(x)), file=filehandle)
+    print('FILES FOUND ARE: %s' % os.listdir('.'), file=filehandle)
+    filehandle.close()
+    sys.exit()
+
+statusfilename = os.path.join(sharedoutputpath,'__jobstatus__')
+statusfile=open_file(statusfilename)
+
+if Backend == "BATCH":
+    heartbeatfilename = os.path.join(sharedoutputpath,'__heartbeat__')
+    heartbeatfile=open_file(heartbeatfilename)
+
+
+line='START: '+ time.strftime('%a %b %d %H:%M:%S %Y',time.gmtime(time.time())) + os.linesep
+if Backend == "BATCH":
+    try:
+        line+='PID: ' + os.getenv('###JOBIDNAME###') + os.linesep
+        line+='QUEUE: ' + os.getenv('###QUEUENAME###') + os.linesep
+        line+='ACTUALCE: ' + hostname() + os.linesep
+    except:
+        pass
+
+statusfile.writelines(line)
+flush_file(statusfile)
+
+if Backend == "LOCAL":
+    workdir = ###WORKDIR###
+    if not os.path.exists(workdir):
+        os.makedirs(workdir)
+    os.chdir(workdir)
+
+
+for f in input_sandbox:
+    if mimetypes.guess_type(f)[1] in ['gzip', 'bzip2']:
+        getPackedInputSandbox(f)
+    else:
+        shutil.copy(f, os.path.join(os.getcwd(), os.path.basename(f)))
+
+# -- END OF MOVED CODE BLOCK
```

Comment:
Can this just be deleted?

Reply:
![ganga2](https://user-images.githubusercontent.com/31144462/79953980-4d019200-849a-11ea-8b25-3d4f08a2c437.png)
 Tried deleting this. Unable to solve this error.